docs/source/test_network.md

```diff
@@ -539,11 +531,8 @@ below provide a guided tour of what happens when you issue the command of
 
 - If you use the `createChannel` subcommand, `./network.sh` runs the
   `createChannel.sh` script in the `scripts` folder to create a channel
-  using the supplied channel name. The script uses the `configtx.yaml` file to
-  create the channel creation transaction, as well as two anchor peer update
-  transactions. The script uses the peer cli to create the channel, join
-  ``peer0.org1.example.com`` and ``peer0.org2.example.com`` to the channel, and
-  make both of the peers anchor peers.
+  using the supplied channel name. The script uses the `configtxgen` tool to create the channel genesis block
+  based on the `TwoOrgsApplicationGenesis` channel profile in the `configtx/configtx.yaml` file. If you do not specify a channel name, the genesis block is named `mychannel.block` and is stored in the `channel-artifacts` folder. The  genesis block is then passed to the `osnadmin` command, which creates the channel. Finally, the script uses the peer cli to join ``peer0.org1.example.com`` and ``peer0.org2.example.com`` to the channel, and make both of the peers anchor peers.
```

Comment:
This strikes me like a lot of information for this basic tutorial. I think the details of how the channel creation works can be handles in later tutorials, and the profile name and different tools can be discussed later.

Reply:
Per our discussion on our workgroup call this morning I simplified this description.